src/gpodder/schema.py

```diff
@@ -194,6 +199,9 @@ def upgrade(db, filename):
         initialize_database(db)
         return
 
+    db.create_function('is_html', 1, util.is_html)
+    db.create_function('remove_html_tags', 1, util.remove_html_tags)
```

Comment:
I have confirmed `util.remove_html_tags` has the same logic as is used by **podcastparser** so no risk of inconsistency there.